evaluation/utils/computer.py

```diff
@@ -203,6 +200,9 @@ def find_next_ready_job(self) -> Optional[Job]:
                     f"Postpone computation."
                 )
                 self._waiting.append(job)
+                self.update_evaluation_status(
```

Comment:
If we're trying to clean up the evaluation statuses, maybe we should distinguish between when something is in `_waiting` and when something is in `_computing`? Right now when something is in `_waiting` or `_computing` the evaluation status will be `evaluating`. (I think) only when something is in `_computing` is it actually being evaluated, so maybe we can add a new status `computing`.

Reply:
As in maybe `evaluating` can mean it is queued for evaluation whereas  `computing` can mean it is currently being computed i.e. after being in `computing` stage it can either be `failed` or `completed`. Maybe we change `evaluating` to `queued for evaluation` and `computing` can become `evaluating`? 

I see how this might be unnecessarily confusing, so no worries if we don't change it.

Reply:
good points, yeah I think it makes sense to make this change.

Reply:
good points, yeah I think it makes sense to make this change.